core/network/impl/CMakeLists.txt

```diff
@@ -3,6 +3,14 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+add_library(loopback_stream
+    loopback_stream.cpp
+    loopback_stream.hpp
```

Comment:
no need to add hpp

Reply:
Usually it uses for expand scope, for ability of IDE to seeing headers. Additional it one of way for simplifying installation. More info you can get [here](https://stackoverflow.com/questions/36174499/why-add-header-files-into-add-library-add-executable-command-in-cmake).
In my own pet-projects I use other way, and not add headers in **add_library**/**add_executable**. In this project I follow existing code.